### ISSUE TITLE
Buttons sending stop working after update

### ISSUE THREAD

**Issue by Eonus21** [pedroslopez/whatsapp-web.js#1749]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

After latest update, buttorns dont work anymore. Sending returns status ok , but in whatsapp web it looks like a loading message. 
![image](https://user-images.githubusercontent.com/49307215/197064681-a74b39f1-f562-4557-9ddd-681e6853db7f.png)

### Expected behavior

As a user, I expect that buttons will be sended to recipient

### Steps to Reproduce the Bug or Issue

1. Install WW and authorize account
2. Try to send a message with buttons

### Relevant Code

```
const convertedButtons = []
for (let i = 0; i < buttons.length; i++) {
      convertedButtons.push({ body: buttons[i].src })
 }
const button = new Buttons(message.text, convertedButtons)
let result = await client.sendMessage(message.to + "@c.us", button)
```

### Browser Type

Google Chrome

### WhatsApp Account Type

Standard

### Does your WhatsApp account have multidevice enabled?

Yes, I am using Multi Device

### Environment

linux
android
latest WW
node v16

**Comment by danieldwipaska** [comment]
same problem here

**Comment by alechkos** [comment]
the update to v1.18.0 doesn't include the buttons/lists fix PR
you have to install it by yourself
just use `npm i github:pedroslopez/whatsapp-web.js#fix-buttons-list` int the terminal
more info here #1636

**Comment by danieldwipaska** [comment]
> the update to v1.18.0 doesn't include the buttons/lists fix PR you have to install it by yourself just use `npm i github:pedroslopez/whatsapp-web.js#fix-buttons-list` more info here #1636

I have tried that. it still doesn't work. When I did that, another problem happened. 'message' event couldn't not triggered.

**Comment by esteban020987** [comment]
> la actualización a v1.18.0 no incluye los botones/listas arreglar PR tienes que instalarlo tú mismo solo usa `npm i github:pedroslopez/whatsapp-web.js#fix-buttons-list` más información aquí #1636

También hice lo mismo, y tampoco funcionó

**Comment by NonameASecret** [comment]
> > the update to v1.18.0 doesn't include the buttons/lists fix PR you have to install it by yourself just use `npm i github:pedroslopez/whatsapp-web.js#fix-buttons-list` more info here #1636
> 
> I have tried that. it still doesn't work. When I did that, another problem happened. 'message' event couldn't not triggered.

**yes,same issue for me**

**Comment by NonameASecret** [comment]
> > > the update to v1.18.0 doesn't include the buttons/lists fix PR you have to install it by yourself just use `npm i github:pedroslopez/whatsapp-web.js#fix-buttons-list` more info here #1636
> > 
> > 
> > I have tried that. it still doesn't work. When I did that, another problem happened. 'message' event couldn't not triggered.
> 
> **yes,same issue for me**

@PurpShell why `fix-buttons-list` (#1636 ) Branch  'message' event not triggered ?

**Comment by jllerenaa** [comment]
> > > > the update to v1.18.0 doesn't include the buttons/lists fix PR you have to install it by yourself just use `npm i github:pedroslopez/whatsapp-web.js#fix-buttons-list` more info here #1636
> > > 
> > > 
> > > I have tried that. it still doesn't work. When I did that, another problem happened. 'message' event couldn't not triggered.
> > 
> > 
> > **yes,same issue for me**
> 
> @PurpShell why `fix-buttons-list` (#1636 ) Branch 'message' event not triggered ?

I have exactly the same Issue, `message` event is not being triggered from yesterday 2022-10-20 on `fix-buttons-list` branch

**Comment by esteban020987** [comment]
> > > > > la actualización a v1.18.0 no incluye los botones/listas arreglar PR tienes que instalarlo tú mismo solo usa `npm i github:pedroslopez/whatsapp-web.js#fix-buttons-list`más información aquí #1636
> > > > 
> > > > 
> > > > He intentado eso. todavía no funciona Cuando hice eso, ocurrió otro problema. El evento 'mensaje' no pudo desencadenarse.
> > > 
> > > 
> > > **si, el mismo problema para mi**
> > 
> > 
> > @PurpShell¿Por qué `fix-buttons-list`( #1636 ) el evento 'mensaje' de la rama no se activó?
> 
> Tengo exactamente el mismo problema, el `message`evento no se activa desde ayer 2022-10-20 en la `fix-buttons-list`rama

Alguien que sepa como corregir ésto?

**Comment by mochamadwiby** [comment]
> > > > the update to v1.18.0 doesn't include the buttons/lists fix PR you have to install it by yourself just use `npm i github:pedroslopez/whatsapp-web.js#fix-buttons-list` more info here #1636
> > > 
> > > 
> > > I have tried that. it still doesn't work. When I did that, another problem happened. 'message' event couldn't not triggered.
> > 
> > 
> > **yes,same issue for me**
> 
> @PurpShell why `fix-buttons-list` (#1636 ) Branch 'message' event not triggered ?

yup

**Comment by esteban020987** [comment]
Corregí el problema de los botones y listas agregando nuevamente la parte de botones (#1636 ) en el archivo Injected.js

**Comment by alechkos** [comment]
> > the update to v1.18.0 doesn't include the buttons/lists fix PR you have to install it by yourself just use `npm i github:pedroslopez/whatsapp-web.js#fix-buttons-list` more info here #1636
> 
> I have tried that. it still doesn't work. When I did that, another problem happened. 'message' event couldn't not triggered.

temporary solution
https://discord.com/channels/698610475432411196/1033022800367202364/1033230333988044810

**Comment by Eonus21** [comment]
Resolved with https://github.com/pedroslopez/whatsapp-web.js/pull/1636. I hope this changes will be merged into main branch.
